zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -53,7 +56,8 @@ pub(crate) struct ConstraintBuilder<'a, F> {
     execution_state: ExecutionState,
     constraints: Vec<(&'static str, Expression<F>)>,
     lookups: Vec<Lookup<F>>,
-    row_usages: Vec<StepRowUsage>,
+    curr_row_usages: Vec<StepRowUsage>,
+    next_row_usages: Vec<StepRowUsage>,
```

Comment:
I am not sure whether this design will be OK. This step will treat state cells in next step as it like, while how the next step treats its own state cells (already constrained) may not be compatible will its previous step.    Maybe next step needs to know that its state cells already consumed by its previous step so it can use constrained cells?  I will read more related codes and comment later.